codebuild/cd/alpine-arm64-build.yml

```diff
@@ -0,0 +1,22 @@
+version: 0.2
```

Comment:
trivial: names these files "musl" or "linux_musl", not ~alpine~. The fact that we're building for "musl" won't change. The detail of using "alpine" might change

Reply:
Thanks, I have updated the names to musl-*.